### DISCUSSION TITLE
Update documentation

### DISCUSSION THREAD

**Discussion by Drsheppard01** [AppImage/AppImageKit#1343] [Q&A] [answered]
Can you update documentation how to build AppImage v3 with static tools, runtime and other things for non-glibc system like @chimera-linux

**Comment by probonopd** [comment]
There is no "AppImage v3". If you use appimagetool from https://github.com/AppImage/appimagetool/releases rather than from https://github.com/AppImage/AppImageKit/releases, then you will get the static runtime.

Some higher-level tools (such as linuxdeployqt) use appimagetool internally; those tools will need to be updated to use the new appimagetool.

However, if you want to create an AppImage that can run on non-glibc systems (or build the ingredients for an AppImage on a non-glibc system), then you need to bundle _everything_ inside the AppImage. Tools like [go-appimage's appimagetool](https://github.com/probonopd/go-appimage/tree/master/src/appimagetool#appimagetool) can do that.

**Comment by Korne127** [comment, marked as answer]
Just for the record, https://github.com/AppImage/docs.appimage.org/pull/102 updates the documentation to go over the different AppImage versions, including the new static runtime and its differences to the old dynamic runtime.
